modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -309,6 +311,10 @@ public Swift5ClientCodegen() {
             "[WARNING] This option will be removed and enabled by default in the future once we've enhanced the code to work with `Data` in all the different situations. Map File and Binary to Data (default: false)")
             .defaultValue(Boolean.FALSE.toString()));
 
+        cliOptions.add(new CliOption(USE_CUSTOM_DATE_WITHOUT_TIME,
+            "[WARNING] This option will be removed and enabled by default in the future. Uses a custom type to de- and encode dates without time information (default: false)")
```

Comment:
@Jonas1893 I'm not sure we should have this message here `[WARNING] This option will be removed and enabled by default in the future.`
Let's first see how it works, then if all goes ok, we can switch the default value to true, and then we can maybe remove this? Or simply leave this, for an easier migration.
So I think that we should remove that part of the docs.

Reply:
Sure, changed